scipy/stats/_stats_py.py

```diff
@@ -8139,44 +8150,67 @@ def combine_pvalues(pvalues, method='fisher', weights=None):
 
     Notes
     -----
-    Fisher's method (also known as Fisher's combined probability test) [1]_ uses
-    a chi-squared statistic to compute a combined p-value. The closely related
-    Stouffer's Z-score method [2]_ uses Z-scores rather than p-values. The
-    advantage of Stouffer's method is that it is straightforward to introduce
-    weights, which can make Stouffer's method more powerful than Fisher's
-    method when the p-values are from studies of different size [6]_ [7]_.
-    The Pearson's method uses :math:`log(1-p_i)` inside the sum whereas Fisher's
-    method uses :math:`log(p_i)` [4]_. For Fisher's and Pearson's method, the
-    sum of the logarithms is multiplied by -2 in the implementation. This
-    quantity has a chi-square distribution that determines the p-value. The
-    `mudholkar_george` method is the difference of the Fisher's and Pearson's
-    test statistics, each of which include the -2 factor [4]_. However, the
-    `mudholkar_george` method does not include these -2 factors. The test
-    statistic of `mudholkar_george` is the sum of logisitic random variables and
-    equation 3.6 in [3]_ is used to approximate the p-value based on Student's
-    t-distribution.
+    All methods assume tests based on continuous distributions, i.e., the
+    distribution of p-values under the null hypothesis must be the continuous
+    uniform distribution on the interval [0,1]. If this function is applied to
+    tests with a discrete statistics such as any rank test or contingency-table
```

Comment:
This brings up a subtlety. Among contingency table tests, something like Fisher's exact test is very obviously discrete, but in principle, a 𝝌² test for contingency tables is based upon a continuous distribution, but for smaller sample sizes the distribution of p-values is discrete because the data is discrete. Since your not trying to be exhaustive with your examples maybe just use a more obviously discrete family of tests like permutation tests of which Fisher's exact test is also an example.

```suggestion
    tests with a discrete statistics such as any rank test or permutation
```

Reply:
I would leave this at contingency tables since they are the more important example in application, discrete in the respect that matters here, and easily discernible as discrete by the target audience for this information. Whether some continuous distribution is used somewhere under the hood (e.g., as an estimator) does not affect this. Also, all tests approximate continuity for a sufficiently large sample size.

Reply:
Yeah. This suggestion isn't strict. It's OK to disregard. As someone trained as a mathematician, I'm kind of a stickler about precise language. It's a matter of how one defines *based upon a continuous distribution.* Is it that the test statistic follows a continuous distribution in theory, or that the statistics available in practice take values in something approximating a continuum? Clearly you're using the second definition, and hopefully people will find it clear from the context. Also, I guess deep down everything in computational statistics is discrete.